### PR TITLE
Remove code related to tobs CLI

### LINKED ISSUE #396: Remove tobs CLI code
To increase development velocity and align with common best practices for Continuous Delivery (and GitOps) we decided to remove tobs CLI and focus on development of the helm chart as well as components shipped by said helm chart.

### PR DESCRIPTION
**Pull request by onprem**
## Description

Fixes #396 

## Type of change

*What type of changes does your code introduce to tobs? Put an `x` in the box that apply.*

- [x] `CHANGE` (fix or feature that would cause existing functionality to not work as expected)
- [ ] `FEATURE` (non-breaking change which adds functionality)
- [ ] `BUGFIX` (non-breaking change which fixes an issue)
- [ ] `ENHANCEMENT` (non-breaking change which improves existing functionality)
- [ ] `NONE` (if none of the other choices apply. Example, tooling, build system, CI, docs, etc.)

### CODE REVIEW

File: .github/workflows/tests.yml [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+      - name: Wait for cluster to finish bootstraping
+        run: kubectl wait --for=condition=Ready pods --all --all-namespaces --timeout=300s
+
+      - name: Run chart-testing (install)
+        run: ct install --config ct.yaml --upgrade
+
+      # TODO(onprem): Figure out a workaround to test datasources as ct just removes
+      # the chart release after a successful install.
+      # - name: Check datasources
+      #   run: make check-datasources
```
Comment (onprem) [inline .github/workflows/tests.yml:55]: Maybe a separate GH Action workflow with `helm install` followed by `make check-datasources`?

File: .github/workflows/tests.yml [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+        env:
+          KUBE_VERSION: ${{ matrix.kube-version }}
+        run: |
+          make cert-manager
+
+      - name: Wait for cluster to finish bootstraping
+        run: kubectl wait --for=condition=Ready pods --all --all-namespaces --timeout=300s
+
+      - name: Run chart-testing (install)
+        run: ct install --config ct.yaml --upgrade
```
Comment (onprem) [inline .github/workflows/tests.yml:50]: Still need to confirm this but there is a problem here. 

The `ct install` command, uninstalls the chart after everything is up and running and then waits for every resource to get deleted in the namespace it created. But some components don't go away even after a uninstall (like OTel Collector as it's a hook, and some secrets).

This means that `ct install` will get stuck and then timeout and fail, always.

One workaround is using a background job to delete the resources using kubectl once they are in up and running state.

File: .github/workflows/tests.yml (hunk 1 above)
Comment (paulfantom) [reply .github/workflows/tests.yml:55]: SGTM

**Comment by paulfantom** [comment]
It seems like the issue might be with github actions not having enough resources to run the whole stack:

>   Warning  FailedScheduling  6s (x5 over 67s)  default-scheduler  0/1 nodes are available: 1 Insufficient cpu. preemption: 0/1 nodes are available: 1 No preemption victims found for incoming pod.

Could you test this theory by temporarily removing resource requests from prometheus, promscale, and timescaledb pods (done in `chart/values.yaml`)

If that is the case, we can create `values.yaml` patches and place them in `chart/ci/XYZ-values.yaml` files as per [`ct install docs`](https://github.com/helm/chart-testing/blob/main/doc/ct_install.md#synopsis.):

> Charts may have multiple custom values files matching the glob pattern '*-values.yaml' in a directory named 'ci' in the root of the chart's directory. The chart is installed and tested for each of these files. If no custom values file is present, the chart is installed and tested with defaults.

**Comment by onprem** [comment]
Waiting on #493 as that will fix the timeout issue with `ct install`.

**Comment by onprem** [comment]
> It seems like the issue might be with github actions not having enough resources to run the whole stack:
> 
> > Warning  FailedScheduling  6s (x5 over 67s)  default-scheduler  0/1 nodes are available: 1 Insufficient cpu. preemption: 0/1 nodes are available: 1 No preemption victims found for incoming pod.
> 
> Could you test this theory by temporarily removing resource requests from prometheus, promscale, and timescaledb pods (done in `chart/values.yaml`)
> 
> If that is the case, we can create `values.yaml` patches and place them in `chart/ci/XYZ-values.yaml` files as per [`ct install docs`](https://github.com/helm/chart-testing/blob/main/doc/ct_install.md#synopsis.):
> 
> > Charts may have multiple custom values files matching the glob pattern '*-values.yaml' in a directory named 'ci' in the root of the chart's directory. The chart is installed and tested for each of these files. If no custom values file is present, the chart is installed and tested with defaults.

Looks like the case to me as well. We already have a values.yaml there in this PR to disable telemetry. I'll remove the resource requests and then test.

**Comment by onprem** [comment]
Now the CI is failing because `ct` deletes the namespace in the end, with a hardcoded timeout of `180s`. But with PVCs and finalisers, tobs almost always takes longer than `180s` to delete the namespace, causing `ct install` to fail.

Upstream issue: helm/chart-testing#227

File: .github/workflows/tests.yml [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+      - name: Wait for cluster to finish bootstraping
+        run: kubectl wait --for=condition=Ready pods --all --all-namespaces --timeout=300s
+
+      - name: Create namespace for installing the chart
+        run: kubectl create namespace tobs-test
+
+      - name: Run chart-testing (install)
+        run: ct install --config ct.yaml --namespace tobs-test
+
+      # TODO(onprem): Figure out a workaround to test datasources as ct just removes
```
Comment (onprem) [inline .github/workflows/tests.yml:55]: Will do a follow up PR for this.

File: RELEASE.md [hunk 4]
```diff
@@ -36,10 +34,6 @@ For a new major or minor release, work from the `master` branch. For a patch rel
 
 Update version information in multiple places:
 - `version` and `appVersion` in `chart/Chart.yml`
-- `tobsVersion` in `cli/cmd/version/version.go`
-- `TOBS_VERSION` in `install-cli.sh`
-
-Next, run `cp chart/values.yaml cli/tests/testdata/e2e-values.yaml` to sync testing data and remember to re-apply code part responsible for disabling telemetry (as seen in [PR#412](https://github.com/timescale/tobs/pull/412/files)).
```
Comment (onprem) [inline RELEASE.md:42]: Need to add similar instructions for `chart/ci/e2e-values.yaml`

**Review by nhudson — APPROVED** [review]
LGTM 🚀

**Pull request merged**